### DISCUSSION TITLE
Notification: Ability to just subscribe contributors and repository's activities

### DISCUSSION THREAD

**Discussion by vuon9** [community/community#6283] [Projects and Issues]
As a user, I want to avoid being distracted by getting new comments which has no value from many people in my subscribed issue, and definitely that I don't want to unsubscribe the issue. 

It would be nice if I can have a notification option for only activities of contributors and repository in the issue. Additionally, It could be so cool if I can also subscribe if I see valuable people in the topic by tagging their usernames in Notification setting as well.

Here is a terrible mockup of what I explained.
<img width="462" alt="Screenshot 2021-10-13 at 6 31 11 PM" src="https://user-images.githubusercontent.com/3168632/137124741-a6daa4d2-95c8-4f45-b997-55124edf8546.png">

**Comment by jrr** [comment]
I feel this all the time. In any sufficiently-large project, issues attract a lot of unhelpful spam. I wind up unsubscribing, even if I still want to know when the problem is fixed.

I'd be happy with any improvement made here, even if it isn't as sophisticated as @vuon9 's mockup. At minimum, "notify me when this issue is closed" would be a great start.
